Automattic/jetpack
# Sync: Send updated plugins info after updating via the API

#### Changes proposed in this Pull Request:
Try to sync plugins data right after a plugin update API call

#### Testing instructions:

* Edit and downgrade a plugin(akismet on the examples) version so it needs an update
* Verify that `http GET https://public-api.wordpress.com/rest/v1.2/sites/example.blog/plugins/akismet  'Authorization: BEARER <your token>'` returns the changed version
* Update the plugin via `http POST https://public-api.wordpress.com/rest/v1.2/sites/example.blog/plugins/akismet%2F/update  'Authorization: BEARER <your token>'`
* Verify that `http GET https://public-api.wordpress.com/rest/v1.2/sites/example.blog/plugins/akismet  'Authorization: BEARER <your token>'` returns the new version


@oskosk this is a bug, it would be great to add it to the next beta :) Thanks 🙇
